conf/blocklist/sync-fail2ban

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+##  Update fail2ban iptables with globally known attackers.
+##  Actually, runs 100% independently now, without needing fail2ban installed.
+##
+##  /etc/cron.daily/sync-fail2ban
+##
+## Author: Marcos Kobylecki <fail2ban.globalBlackList@askmarcos.com>
+## http://www.reddit.com/r/linux/comments/2nvzur/shared_blacklists_from_fail2ban/
+
+
+## Quit if fail2ban is missing.  Maybe this fake requirement can be skipped? YES.
+#PROGRAM=/etc/init.d/fail2ban
+#[ -x $PROGRAM ] || exit 0
+
+datadir=/etc/fail2ban
+[[ -d "$datadir" ]] || datadir=/tmp
+
+## Get default settings of fail2ban (optional?)
+[ -r /etc/default/fail2ban ] && . /etc/default/fail2ban
+
+umask 000
+blacklistf=$datadir/blacklist.blocklist.de.txt
+
+mv -vf  $blacklistf  $blacklistf.last
+
+badlisturls="http://antivirus.neu.edu.cn/ssh/lists/base_30days.txt http://lists.blocklist.de/lists/ssh.txt  http://lists.blocklist.de/lists/bruteforcelogin.txt"
+
+
+ iptables -vN fail2ban-ssh   # Create the chain if it doesn't exist. Harmless if it does.
+  
+# Grab list(s) at https://www.blocklist.de/en/export.html .  Block.
+echo "Adding new blocks:"
+ time  curl -s http://lists.blocklist.de/lists/ssh.txt  http://lists.blocklist.de/lists/bruteforcelogin.txt \
```

Comment:
The exit status of curl isn't checked. If the remote server is down, the empty result set is processed I believe.